R/MetAlyzer-seclass.R

```diff
@@ -294,10 +232,9 @@ summarizeQuantData <- function(metalyzer) {
   cat(paste0("NAs: ", nas, " (", round(nas / total * 100, 2), "%)\n"))
   status_list[["NA"]] <- status_metabolites(quant_status, "NA")
   cat("-------------------------------------\n")
-  return(status_list)
+  return(status_list) ### why ???
```

Comment:
The `status_list` holds a list of metabolites for each quantification status. If two metabolites have the status `Invalid` they can be removed from the dataset.